client/components/domains/register-domain-step/index.jsx

```diff
@@ -586,6 +588,10 @@ class RegisterDomainStep extends React.Component {
 			return;
 		}
 
+		if ( searchQuery.length < MIN_QUERY_LENGTH ) {
```

Comment:
You should probably do this check after calling [`getFixedDomainSearch`](https://github.com/Automattic/wp-calypso/blob/34c805a5a254ad0a425f16d6e9ef90fd015d05e1/client/lib/domains/index.js#L166) - it trims the query, among other things. Otherwise query like `a ` _will_ trigger a search (but the query passed to the backend will be `a`, which could lead to some confusing stats or events).

Reply:
Eh, you make my job easy. :)